frontend/src/components/layout/Layout.module.scss

```diff
@@ -124,6 +127,7 @@ $toastify-toast-close-button: var(--toast-close-button);
 #close-toast-button-icon {
   width: 20px;
   height: 20px;
+  min-width: 20px;
```

Comment:
question: Is this necessary? I imagine the static width attr does enough?

Reply:
The icon keeps getting smaller the smaller your screen is. Regardless of width given.